src/renderer/components/+catalog/catalog.tsx

```diff
@@ -144,7 +146,11 @@ export class Catalog extends React.Component<Props> {
   };
 
   renderNavigation() {
-    return <CatalogMenu activeItem={this.activeTab} onItemClick={this.onTabChange}/>;
+    return (
+      <RenderDelay>
+        <CatalogMenu activeItem={this.activeTab} onItemClick={this.onTabChange}/>
```

Comment:
I noticed this flashes in the videos you show, maybe this shouldn't be delayed

Reply:
Yes, but this component affect performance. Probably we should go away from Material `<TreeView/>` for sidebar. Some css-in-js operations affect rendering, as it turned out.